cmd/utils/flags.go

```diff
@@ -130,7 +130,7 @@ var (
 	}
 	NetworkIdFlag = cli.Uint64Flag{
 		Name:  "networkid",
-		Usage: "Network identifier (integer, 1=Frontier, 2=Morden (disused), 3=Ropsten, 4=Rinkeby, 6=Kotti)",
+		Usage: "Network identifier (integer, 1=Frontier, 2=Morden (disused), 3=Ropsten, 4=Rinkeby, 6=Kotti, 77=Astor)",
```

Comment:
Upstream merged goerli, so you have to resolve a lot of conflicts here, unfortunately.